doc/user-guide.md

```
@@ -27,17 +27,17 @@ The panel opens on the right side of the console, automatically focused on what
 
 > [!NOTE]
 > Some pages don't support correlation, for example the _Overview_ page. The panel will show `No results`.
-> Use the `Focus` button to focus on a more interesting page.
+> Use the `Correlate` button to correlate from a more interesting page.
 
-## Focus
+## Correlate
 
 ![](images/toolbar.png)
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)